CHANGELOG-template.rst

```diff
@@ -0,0 +1,35 @@
+0.XX.0
+------
+
+Renku ``0.XX.0`` introduces ...
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- 🐸 **Team**: changelog item with emoji, ordered by importance to users
+
+**✨ Improvements**
+
+- 🐸 **Team**: changelog item with emoji, alphabetized by team name
+
+**🐞 Bug Fixes**
+
+- 🐸 **Team**: changelog item with emoji, alphabetized by team name
```

Comment:
Bug fixes don't have emojis in the previous releases. Is the emoji intended here (i.e. it's going to change in the next releases) or should it rather be removed also from here

Reply:
Oh yes, thank you, fixed!